expression-language/3.0/tck_results_summary.txt

```diff
@@ -0,0 +1,344 @@
+[javatest.batch] Number of Tests Passed      = 336
```

Comment:
The results summary needs to be published on a page related to the implementation project,
not here.